src/main/java/net/sf/jabref/model/groups/GroupsParser.java

```diff
@@ -1,16 +1,15 @@
-package net.sf.jabref.logic.groups;
+package net.sf.jabref.model.groups;
```

Comment:
I'm not sure if it is possible to move it to logic, but I might have a look when rebasing. The main reason (shorttime) to move things like this to logic is that they introduce many dependencies to logic, so it is easier to move one method to logic than many to model. This one seems OK from that perspective if nothing else.